sopel/trigger.py

```diff
@@ -129,7 +132,10 @@ def __init__(self, own_nick, line, url_schemes=None):
         self.time = datetime.datetime.utcnow()
```

Comment:
If the server time is not provided, this would have been a naive datetime, which is bad.

```suggestion
        self.time = pytz.utc.localize(datetime.datetime.utcnow())
```